src/UI/Component/Dropzone/File/Dropzone.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Interface Dropzone
+ *
+ * Describes a dropzone that can be configured to use the darkened background
+ * or not.
```

Comment:
Couldn't a skin decide to use a lightened background or any other CSS instead? I think it might not be a good idea to put this into the naming of an interface. And are we sure that this is something interesting for file dropzones only? I wonder if the hierarchy path Dropzone > File > Dropzone is correct here.